docs/UserGuide.md

```diff
@@ -305,61 +305,81 @@ List all the events stored in the database with their unique index number.
 
 Delete the task with the unique index of `00123`.
 
-### Edit a floating task: `edit`
+### Edit a floating task/event/dealine: `edit`
+
+* Edit command can only edit the parameters of each commands but cannot transform a task to another
+type. For example, `eidt` cannot transform a floating task to a event task.
+
+#### Edit an floating task:
 
 Edit a floating task to revise its name or priority.
 
-    edit "TASK_NAME" [n-NEW_NAME | p-PRIORITY]...
+    edit TASK_UNIQUE_INDEX [n-NEW_NAME | p-PRIORITY]...
 
-* Task name should be in a pair of quotation marks.
+* Quotation marks are not necessary for `NEW_NAME`.
 
 * `PRIORITY` should only be the interger ranges from `0` to `5`.
 
-* Here we assume that no duplicate task name is used among all the task.
-
 #### Examples
 
-* `edit "washing clothes" p-0`
+* `edit 00124 p-0`
```

Comment:
Yes, I agree that this is an improvement over having the user type the name of the tasks.

However, I suggested an even better idea:
> Why not just keep the current behaviour where the index is based on the position of the task in the list in the user interface?

Currently, with our UI design, we only see a small number of tasks at any one time. E.g., we see the events that happen today (which is a small number), we see the deadlines that happen soon (should be a small number). So, with the above, the user will likely only need to type at most 2 digits.

Contrast it with your "unique ID" design, where the user will likely end up having to type 3 digits or more once the user starts storing over a hundred tasks. I'm sure we can agree that my solution will lead to the user having to type *less* digits on average.